modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase

```diff
@@ -15,6 +15,182 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-11779. Fulfill the order from the supplier side, and from the buyer side, you can view the information from the split supplier order in the customer order"
+	@priority = 5
+	test CanAssertStoreFrontSplitOrderShipmentStatuses {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site with a buyer user is created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And a Supplier account with a supplier user is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Commerce Supplier",
+				accountEntryType = "Supplier");
+
+			JSONAccountEntryUser.addAccountEntryUser(
+				accountEntryName = "Commerce Supplier",
+				userEmailAddress = "supplieruser@liferay.com",
+				userFirstName = "Supplier",
+				userLastName = "User",
+				userScreenName = "supplieruser");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "supplieruser@liferay.com");
+
+			JSONAccountRole.associateUser(
+				accountEntryName = "Commerce Supplier",
+				accountRole = "Account Supplier",
+				userEmailAddress = "supplieruser@liferay.com");
+		}
+
+		task ("And the supplier account is linked to Minium Portal Channel") {
+			CommerceJSONChannelsAPI._patchCommerceChannel(
+				accountEntryName = "Commerce Supplier",
+				channelName = "Minium Portal");
+		}
+
+		task ("And a product with 'Master' catalog, a price and an inventory is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Master",
+				price = 25,
+				productName = "Simple Product",
+				productType = "simple",
+				sku = "TESTSKU");
+
+			CommerceJSONWarehousesAndInventoriesAPI._addWarehouseItem(
+				warehouseItemQuantity = 100,
+				warehouseItemSku = "TESTSKU",
+				warehouseName = "Italy");
+		}
```

Comment:
Can you use a new 'Supplier Catalog' and a 'Supplier Channel' instead of using the Master catalog and Minium's channel or creating a 2nd Minium site? Please apply to all tests, thanks!

Reply:
Sorry, I'm not very clear about this, if we use a new catalog and channel, we cannot get the expected results in this test and it might be similar to the other test `CanManageSupplierOrdersOnlyFromBothCatalogAndChannelLinkedToSupplierAccount`, could you help clarify it?